### ISSUE TITLE
Нажаль роботу з списками ще непроходили

### ISSUE THREAD

**Issue by Muxamedin** [Omel1311/Omelianenko_GH#3]
https://github.com/Omel1311/Omelianenko_GH/blob/65084599d8d1cb53c01a44223ab9ff7c1d36f27f/Lesson_5/task_1.py#L9

Що як розлянути такий підхід :
<img width="665" alt="image" src="https://user-images.githubusercontent.com/4419752/183288264-c6531d49-6018-488a-9da4-25e52aaaf940.png">

**Comment by Omel1311** [comment]
Так! Алгоритм в принципі той самий (для вирішення потрібно порівнювати дві змінні). Я щось розумів, ходив навкруги майже добу...Але вдадось вирішити трошки інакше...
Мої "домашні" спроби розуміння алгоритму:

![20220807_144140](https://user-images.githubusercontent.com/109906648/183289046-b8422516-3367-4d40-a302-2e4bd2051e80.jpg)
![20220807_143725](https://user-images.githubusercontent.com/109906648/183289049-5bbdcd7b-e90b-4dce-84aa-e118605dbed0.jpg)
![20220807_144146](https://user-images.githubusercontent.com/109906648/183289052-795e8000-a785-47ac-bbcb-84466a9ca2b4.jpg)

**Comment by Muxamedin** [comment]
тут можна було не переводити до списків 
тому що можна булоб вільно працювати з індексами рядків

**Comment by Omel1311** [comment]
так..повністю згоден! в моєму рішенні є "лишні рухи"..можна було б як у вашому варіанті.
Досвід...
